inc/tinyMce.php

```diff
@@ -81,6 +81,42 @@ function getConfig()
             'Heading 6' => 'h6'
         ],
         'styleformats' => [
+            [
+                'title' => 'Headings',
+                'icon' => '',
+                'items' => [
+                    [
+                        'title' => 'Heading 1',
+                        'classes' => 'h1',
+                        'selector' => 'h1,h2,h3,h4,h5,h6,p'
```

Comment:
Looking great! I've got 2 thoughts I would like your opinion about:

1. Why do you prefer a specific selector, instead of allowing the classes for all elements?
2. I we keep the selector, how about removing the selector`h1` for a Heading 1 `h2` for Heading 2, and so on?

Reply:
I've allowed the heading classes to be added to anything, since I think that does make sense.
The reason I added all of the selectors was that TinyMCE uses the first selector to style the item in the dropdown menu. The result was that "Heading 1" was smaller than the others in the list... We, of course, add the backend styles so this wouldn't be a big issue, but I thought someone using flynt out of the box might find it strange that the largest heading was smaller than the others. Allowing the classes to be given to all elements bypasses this though.

Reply:
Alright, I didn't know that! I agree with your point.